schemas/json/layout/layout.schema.v1.json

```diff
@@ -323,6 +324,54 @@
         }
       }
     },
+    "map": {
+      "properties": {
+        "layers": {
+          "type": "object",
+          "title": "Layers",
+          "description": "Map layer",
+          "required": ["url"],
+          "properties": {
+            "url": {
+              "type": "string",
+              "title": "Map layer url",
+              "description": "Url to the map layer"
+            },
+            "attribution": {
+              "type": "string",
+              "title": "Attribution",
+              "description": "Ascribing a work or remark to a particular unit for recognition"
+            },
+            "subdomains": {
+              "type": "array",
+              "title": "Subdomains"
```

Comment:
```suggestion
              "title": "List of subdomains. Used for balancing the load on different map tiling servers. A random one will replace {s} in the defined url."
```

Reply:
We could probably also have `"items": { "type": "string" }` in here as well, to indicate each array item is supposed to be a string.

Reply:
Great feedback, i keep the title to be Subdomains only but added a description with _"List of subdomains. Used for balancing the load on different map tiling servers. A random one will replace {s} in the defined url."_.

Reply:
Ah, yes, I agree! I didn't notice I put everything in the title.. ;-)